src/tests/pages/CaseAPI/createSummary/createSummaryHearingAttendeesPage.ts

```diff
@@ -70,9 +70,9 @@ const createSummaryHearingAttendeesPage: CreateSummaryHearingAttendeesPage = {
         expect(page.locator("#memberList > div > h2")).toHaveText(
           createSummaryHearingAttendeesContent.subTitle1,
         ),
-        expect(
-          page.locator("#memberList_0_0 > div > div > label > h3"),
-        ).toHaveText(createSummaryHearingAttendeesContent.subTitle1),
+        // expect(
```

Comment:
```suggestion
        expect(
          page.locator("#memberList_0_0 h3"),
        ).toHaveText(createSummaryHearingAttendeesContent.subTitle1),
```
Would this not work instead of removing the assertion?